donate/settings/environment.py

```diff
@@ -0,0 +1,68 @@
+import os
+import environ
+
+# We set defaults for values that aren't security related
+# to the least permissive setting. For security related values,
+# we rely on it being explicitly set (no default values) so that
+# we error out first.
+env = environ.Env(
+    ALLOWED_HOSTS=(list, '*'),
+    ASSET_DOMAIN=(str, 'donate.mofoprod.net'),
```

Comment:
It's not used on Heroku. Are we're using `AWS_S3_CUSTOM_DOMAIN` instead?

Reply:
Hmm I don't know what this is for. I'll dig a little and find out if we depend on it

Reply:
This looks liek a relic from the previous network-api repository merge.